### PR TITLE
ui issues for all themes

### PR DESCRIPTION
**Pull request by maxgrapps**
1) Changed header nav links color, logo size fixed, initial and hover styles for network changer, removed colored line on hover for network changer
was:
Screen Shot 2019-06-05 at 13 44 23
now:
Screen Shot 2019-06-05 at 13 44 48

2) Footer logo size fix, added white color on hover for links and socials
now:
Screen Shot 2019-06-05 at 13 46 47

3) Added some transition for buttons, dropdowns, etc...

4) Removed bold font, added darken shade of gray to titles
was:

now:
Screen Shot 2019-06-05 at 13 48 48

5) Font for "6,721,975Gas Limit" changed to regular
was:

now:
Screen Shot 2019-06-05 at 13 49 37

### CODE REVIEW

File: apps/block_scout_web/assets/css/components/_tile.scss [hunk 1]
```diff
@@ -40,11 +40,12 @@ $tile-body-a-color: #5959d8 !default;
     }
 
     .tile-status-label {
-      @if ($label-color) {
-        color: $label-color;
-      } @else {
-        color: $color;
-      }
+      // @if ($label-color) {
```
Comment (vbaranov) [inline apps/block_scout_web/assets/css/components/_tile.scss]: @maxgrapps let's remove commented code

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,6 +35,7 @@
 - [#2064](https://github.com/poanetwork/blockscout/pull/2064) - feat: add fields to tx apis, small cleanups
 
 ### Fixes
+- [#2103](https://github.com/poanetwork/blockscout/pull/2103) - ui issues for all themes
```
Comment (vbaranov) [inline CHANGELOG.md]: @maxgrapps we already issued 2.0.0 release please move this row to the "Current" section

File: apps/block_scout_web/assets/css/components/_btn_dropdown_line.scss [hunk 3]
```diff
@@ -1,15 +1,15 @@
 $btn-dropdown-line-bg: #fff !default;
-$btn-dropdown-line-color: #333 !default;
+$btn-dropdown-line-color: #e2e5ec !default;
 
 .btn-dropdown-line {
   @include btn-line($btn-dropdown-line-bg, $btn-dropdown-line-color);
   border-color: $btn-dropdown-line-color;
-  color: $btn-dropdown-line-color;
+  color: #333;
```
Comment (vbaranov) [inline apps/block_scout_web/assets/css/components/_btn_dropdown_line.scss]: @maxgrapps let's define in var

**Comment by maxgrapps** [comment]
@vbaranov fixed, thanks

**Pull request merged**